internal/db/db.go

```diff
@@ -22,6 +22,7 @@ type ComposeEntry struct {
 type DB interface {
 	InsertCompose(jobId, accountId, orgId string, request json.RawMessage) error
 	GetComposes(accountId string, limit, offset int) ([]ComposeEntry, int, error)
+	DoesComposesBelongsToAccount(accountId string, composeId string) (bool, error)
```

Comment:
How about `GetCompose (accountId string, composeId uuid) (*ComposeEntry, error)` ? Less logic in the db that way, and might be useful in future. 

And then define an error which can be used to distinguish between 'real errors' and not found.

```
type NoRowsError struct {}

func (err *NoRowsError) Error() string {
    return "No rows found"
}
``` 
Also see https://go.dev/blog/go1.13-errors.

Reply:
I was quite happy with the logic right there, but you proposition is nice too.

Reply:
I'm not entirely against having a specific function which just checks if there was an element found, but in that case `ComposeBelongsToAccount` or `DoesComposeBelongToAccount` is correcter.

Reply:
Oh yes, but I'm already implementing your proposition, but thanks for the correction for the name, it was indeed a bit off.

Reply:
I went for a go 1.13 error definition:

`var ComposeNotFoundError = errors.New("Compose not found")`